packages/eslint-plugin-orbit-internal/src/configs/recommended.ts

```diff
@@ -0,0 +1,12 @@
+const recommended = {
+  plugins: ["orbit-internal"],
+  rules: {
+    "orbit-internal/unique-id": "error",
+  },
+  parserOptions: {
+    sourceType: "module",
+    ecmaVersion: 2018,
+  },
```

Comment:
It doesn't seem like it should be the concern of this plugin's recommended config to set this. It might overwrite settings of a plugin before it.